packages/vue/examples/search-showcase/ai-answer-simple/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vue.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Reactivesearch | Searchbox inline ai response</title>
```

Comment:
@SavvyShah The title is misleading, it should be something like "AIAnswer simple"